vantage6-node/vantage6/node/k8s/container_manager.py

```diff
@@ -201,31 +201,66 @@ def _setup_policies(self, config: dict) -> dict:
             )
         return policies
 
-    def _set_database(self, config_databases: dict | list) -> dict:
+    def _get_database_metadata(self) -> dict:
         """
-        Set database location and whether or not it is a file
+        Collect information about the databases.
 
-        Parameters
-        ----------
-        databases: dict | list
-            databases as specified in the config file
+        This function will return a dictionary with contains a key for every data
+        source. Each key will have a dictionary with the following keys:
+        - uri: the URI of the database
+        - local_uri: the local URI of the database
+        - is_file: whether the database is a file
+        - is_dir: whether the database is a directory
+        - type: the type of the database
+        - env: the environment variables to be passed to the container
+
+        It reads these from the environment variables that are passed to the container.
```

Comment:
Nice that you extended this docstring!
```suggestion
        It reads these from the environment variables that are passed to the node.
```

Reply:
955ff2bf36eaba974a573ddb95b9a27548227ff3